components/kyma-environment-broker/internal/process/upgrade_kyma/upgrade_kyma_step.go

```diff
@@ -32,7 +42,7 @@ func (s *UpgradeKymaStep) Name() string {
 }
 
 func (s *UpgradeKymaStep) Run(operation internal.UpgradeKymaOperation, log logrus.FieldLogger) (internal.UpgradeKymaOperation, time.Duration, error) {
-	if time.Since(operation.UpdatedAt) > UpgradeKymaTimeout {
```

Comment:
IMO `UpgradeKymaTimeout` field doesn't really fit in the `IntervalConfig` struct, maybe we can call it `TimeSchedule` or `OperationSchedule`? Anyway if we want it to be parametrized we can get rid of the `UpgradeKymaTimeout` const from that file: https://github.com/kyma-project/control-plane/blob/master/components/kyma-environment-broker/internal/process/upgrade_kyma/upgrade_kyma_step.go#L16.